en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md

```diff
@@ -506,11 +506,30 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
 
 7. Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.
 
-8. Add below configuration  to `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` to disable group role separation. 
-   
+    !!! important
+            In WSO2 Identity Server 5.11.0, groups include user store roles and roles include internal roles. Therefore, from IS 5.11.0 onwards, we cannot have the same admin role in both primary and internal user domains. If the same admin role exists in both UM domains of your older version, you have to rename the internal admin role into a different role name. To do that, you have to follow the below steps.
```

Comment:
AFAIK, with the fix given by IS, we didnt rename the internal admin, what it does is, they will add the primary/admin in to internal domain with given admin role in the toml configuration (admin_role = "<NEW-ADMIN-ROLE-NAME>") during the group role separation. @binodmx Am I correct or wrong?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6092/commits/ecd158325600f976aae1192b6b7ae465d34cd5a8